tests/Controllers/LoginTest.php

```diff
@@ -93,6 +93,8 @@ public function testLoginActionEmailSuccess(): void
 
         // Session should have `logged_in` value with user's id
         $this->assertSame($this->user->id, session('user')['id']);
+
+        Time::setTestNow();
     }
 
     public function testAfterLoggedInNotDesplayLoginPage(): void
```

Comment:
```suggestion
    public function testAfterLoggedInNotDisplayLoginPage(): void
```

Reply:
@kenjis this is my misspelling, please correct it if possible.

Reply:
Okay. I'll do it.

Reply:
Done.